selendroid-server-common/src/main/java/io/selendroid/server/common/http/HttpServer.java

```diff
@@ -69,6 +69,7 @@ public void stop() {
       throw new IllegalStateException("Server is not running");
     }
     serverThread.interrupt();
+    TrafficCounter.shutdown();
```

Comment:
Should this be called in `finally` above instead? The `TrafficCounter` is set up in the `ServerInitializer`.